source/Calamari/Program.cs

```diff
@@ -67,10 +67,12 @@ protected override void ConfigureContainer(ContainerBuilder builder, CommonOptio
             builder.RegisterType<CalamariCertificateStore>().As<ICertificateStore>().SingleInstance();
             builder.RegisterType<DeploymentJournalWriter>().As<IDeploymentJournalWriter>().SingleInstance();
             builder.RegisterType<PackageStore>().As<IPackageStore>().SingleInstance();
+#if !NET40
```

Comment:
Just not compiling stuff that can never be used by .net 4.0 versions of Calamari as it means we can reference Tasks/async without it failing on the .net 4.0 build.